src/main/java/graphql/ExecutionResult.java

```diff
@@ -11,6 +11,13 @@
 
     List<GraphQLError> getErrors();
 
-    Map<Object,Object> getExtensions();
+    Map<Object, Object> getExtensions();
```

Comment:
@bbakerman yes, this is what I meant: it is reserved for the server implementation do something with it and currently we don't use it. And on top of that it is just strange to have such a map, but it never used.

@guy120494 Thanks for your feedback. I agree it is a separate PR.